librz/arch/meson.build

```diff
@@ -307,19 +307,33 @@ if capstone_dep.version() == 'next' or capstone_dep.version().split('.')[0].to_i
   arch_plugins_list += [
     'riscv_cs',
     'tricore_cs',
-    'xtensa_cs',
   ]
 
   # plugins sources
   arch_plugin_sources += [
     'p/arch_riscv_cs.c',
     'p/arch_tricore_cs.c',
-    'p/arch_xtensa_cs.c',
   ]
 
   # isa sources
   arch_isa_sources += [
     'isa/tricore/tricore_il.c',
+  ]
+endif
+
+if capstone_dep.version() == 'next'
```

Comment:
Maybe also check if it's `>=6`? Since 6.0 release will be relatively soon

Reply:
Maybe waiting for the v6 update it would work, but adding v6 in now still doesn't pass the CI.
I think it's best to leave it like this for now and maybe change it in a few days.